src/Compute/Compute/ChangeLog.md

```diff
@@ -28,6 +28,7 @@
 * Updated Set-AzVMRunCommand and Set-AzVmssRunCommand ScriptLocalPath parameter set to work with Linux and with files that have comments.
 * Added `-TargetExtendedLocation` parameter to `New-AzGalleryImageVersion` and `Update-AzGalleryImageVersion`
 * Added `-AllowDeletionOfReplicatedLocation` to `Update-AzGalleryImageVersion`
+* Added ''MaxSurge' to Set-AzVmssRollingUpgradePolicyCommand 
```

Comment:
Please insert your change log message between Line22 and Line23 so that we can make sure your changes included in next release.

Reply:
added, thanks.